### ISSUE TITLE
Add spatial grid

### ISSUE THREAD

**Issue by dfalster** [traitecoevo/australian_annuals#2]
Need a basic mesh/frid across Australia, dividing it into cells, that we can use to poison individual records from GBIF in. Ideally each record gets a cell number.  

Please save in folder `data` and add code for loading into `Analysis.qmd`

@yangsophieee  - wondering if you have something like this from your Hons?

**Comment by yangsophieee** [comment]
Yes, please feel free to adapt my code for this! I've linked my data (including GBIF) and code here:
https://unsw-my.sharepoint.com/:f:/g/personal/z5208613_ad_unsw_edu_au/EvHyAHegAphMgBtseG1D2n0B1zoxgXz2W519-fh4Hjye0g?e=cR7pS4

**Comment by wcornwell** [comment]
the science paper used "bioregions" or something instead of a grid.  should discuss which way we want to go.

**Comment by tmesaglio** [comment]
im not a fan of bioregions personally, way too coarse and arbitrary
